index.js

```diff
@@ -392,10 +392,6 @@ Osm.prototype.query = function (bbox, opts, cb) {
 
   var filter = through.obj(function (row, _, next) {
```

Comment:
Feel free to fully remove this `filter` through stream from the pipeline ^__^

Reply:
Yeah I don't **think** we're using `opts.observations` anywhere? I think originally we used this `query()` method for getting observations for MapFilter in Mapeo Desktop, but once @noffe created the type index, we now use `osm.byType()`. It would be worth checking through the stack to check we're not using this option anywhere though.